lib/config/wdtk-routes.rb

```diff
@@ -38,4 +38,7 @@
 
   get '/help/glossary' => 'help#glossary',
       as: :help_glossary
+  
+   get '/help/environmental_information' => 'help#environmental_information',
```

Comment:
```suggestion
  get '/help/environmental_information' => 'help#environmental_information',
```

Looks like a spurious space has snuck in at the start here.